### PR TITLE
fix(settings): mobile style fixes

### LINKED ISSUE #6899: [FxA-Settings] 2FA “Change” and “Disable” buttons are not correctly aligned on mobile
**Build**
Stage [Train 1.193.0]

**Affected platforms**
Android 9
iOS 13.6

**Steps to reproduce**
1. Navigate to https://accounts.stage.mozaws.net/beta/settings/ and login using an existing account.
2. Enable the 2FA.
3. Check how “Change” and “Disable” buttons are displayed.

**Expected results**
2FA “Change” and “Disable” buttons are correctly aligned compared with the other buttons from the Settings page.

**Actual Results**
- 2FA “Change” and “Disable” buttons are displayed a bit to the right.
- This might be regressed by #6795.

Screenshot_49

┆Issue is synchronized with this [Jira Bug](https://jira.mozilla.com/browse/FXA-2697)
┆Attachments: <a href="https://jira.mozilla.com/secure/attachment/15365/Screenshot_14.png">Screenshot_14.png</a>
┆Issue Number: FXA-2697

### LINKED ISSUE #6917: [FxA-Settings] Input cursor incorrectly displayed on “Verify your email” modal on iOS
**Build**
Stage [Train 1.193.0]

**Affected platforms**
iOS 13.6

**Steps to reproduce**
1. Navigate to https://accounts.stage.mozaws.net/beta/settings/ and login using an existing account that hasn’t been used for a long time.
2. Click on the “Add” button from 2FA section.
3. Insert an invalid 6-chars code.
4. Click on the “Verify” button.

**Expected results**
There are no UI issues.

**Actual Results**
2FA security code input cursor is displayed over the “Verify your email” modal.

**Additional notes**
This issue reproduces only for the 2FA section because the “Enter security code” field is automatically focused when the 2FA enabling process is started.

Screenshot_55

┆Issue is synchronized with this [Jira Bug](https://jira.mozilla.com/browse/FXA-2705)
┆Attachments: <a href="https://jira.mozilla.com/secure/attachment/15367/gif4.gif">gif4.gif</a>
┆Issue Number: FXA-2705

### LINKED ISSUE #7009: [FxA-Settings] The icons are displayed with delay in Bento menu
**Build**
Prod [Train 1.194.0]

**Affected platforms**
Windows 10 x64
Android 9
iOS 13.6

**Steps to reproduce**
1. Navigate to https://accounts.stage.mozaws.net/beta/settings/ and login using an existing account.
2. Click on the Bento menu.
3. Observe how the Firefox Services icons are displayed.

**Expected results**
Firefox Services icons are automatically displayed when the Bento menu is opened.

**Actual Results**
Firefox Services icons from Bento menu are displayed with delay at first opening attempt. 

issue52

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-2738)
┆Issue Number: FXA-2738

### LINKED ISSUE #7014: [FxA-Settings] Hamburger menu and “X” buttons don't have tooltip
**Build**
Prod [Train 1.194.0]

**Affected platforms**
Windows 10 64-bit

**Steps to reproduce**
1. Navigate to https://accounts.stage.mozaws.net/beta/settings/ and login using an existing account.
2. Resize the window until the header navigation menu is displayed.
3. Hover using the mouse cursor over the navigation menu.
4. Click on the navigation menu.
5. Hover using the mouse cursor over the “X” button.

**Expected results**
Both menu buttons successfully display a tooltip.

**Actual Results**
No tooltip is displayed for navigation menu and “X” buttons.

issue53

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-2743)
┆Issue Number: FXA-2743

### LINKED ISSUE #7027: [FxA-Settings] Recovery code error does not disappear after going back and forward
**Build**
Stage [Train 1.194.0]

**Affected platforms**
Windows 10
Android 9
iOS

**Steps to reproduce**
1. Navigate to https://accounts.stage.mozaws.net/beta/settings/ and login using an existing account.
2. Click on the "Add" button from 2FA section.
3. Enter the code received after the QR code was scanned and tap on "Continue".
4. Click on the "Continue" button.
5. Type an invalid recovery code and click the “Finish” button.
6. Click “Back” button.
7. Click on the “continue” button.

**Expected results**
The error is no longer displayed.

**Actual Results**
The error is still displayed and points to an empty field.

issue63

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-2756)
┆Issue Number: FXA-2756

### LINKED ISSUE #7477: [FxA-Settings] Horizontal scrolling on mobile when updating display name
## Description

Unexpected horizontal scrolling on mobile when changing display name.
submitted by: Alex Davis

### Steps to reproduce

Click “Change” to change display name
Select the field to start typing

### Expected result

There’s no horizontal scrolling on mobile

### Actual result

When inputting text in a field, it zooms in and creates a horizontal scroll which makes for a weird experience.

### Environment

iOS 14.2
[rbillings: I was unable to repro on Android]

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-3016)
┆Issue Number: FXA-3016

### LINKED ISSUE #7481: [FxA-Settings] trash icon visible under mobile notification
## Description

The trash icon is visible from under the notification bar which could be confusing since it can overlay with the X to dismiss the notification.
Submitted by: Alex Davis

### Steps to reproduce

Clear steps describing how to reproduce the issue. Providing a URL, screenshot.

### Expected result

Notification should be opaque

### Actual result

Screen Shot 2021-02-09 at 12 07 27 PM

### Environment

iOS 14.2

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-3020)
┆Issue Number: FXA-3020

### PR DESCRIPTION
**Pull request by dannycoates**
fixes #7481
fixes #7477
fixes #7014 
fixes #7009
fixes #6917
fixes #7027 
fixes #6899

### CODE REVIEW

**Comment by meandavejustice** [comment]
z-index issue in header can be fixed by adding `z-10` class to header

https://mozilla.slack.com/archives/CLV3KMZ8B/p1613513789366700?thread_ts=1613503568.358600&cid=CLV3KMZ8B

File: packages/fxa-settings/src/components/AppLayout/index.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ export const AppLayout = ({ children }: AppLayoutProps) => {
           <div
             id="alert-bar-root"
             data-testid="alert-bar-root"
+            className="fixed z-10"
```
Comment (meandavejustice) [inline packages/fxa-settings/src/components/AppLayout/index.tsx:29]: If header ends up getting `z-10`, this should be bumped to `z-20` just to be safe

**Comment by meandavejustice** [comment]
This could be existing issue, but I noticed that the support link icon is misaligned in the mobile header 
image

File: packages/fxa-settings/src/components/HeaderLockup/index.tsx [hunk 2]
```diff
@@ -27,6 +27,11 @@ export const HeaderLockup = () => {
             ? l10n.getString('header-menu-open')
             : l10n.getString('header-menu-closed')
         }
+        title={
```
Comment (meandavejustice) [inline packages/fxa-settings/src/components/HeaderLockup/index.tsx:30]: 👍

**Review by meandavejustice — APPROVED** [review]
2 comments:

- Support icon (we should either address here or create issue)
- z-index of header

**Comment by dannycoates** [comment]
I can't repro those header issues

**Pull request merged**